.travis.yml

```diff
@@ -30,14 +30,21 @@ before_deploy:
   - npm set //registry.npmjs.org/:_authToken "$NPM_TOKEN"
 
 deploy:
-  provider: script
-  script: npm run publish
-  skip_cleanup: true
-  on:
-    tags: true
-    branch: main
-    repo: twilio/flex-plugin-builder
-    node_js: '10'
+  - provider: script
+    script: npm run publish
+    skip_cleanup: true
+    on:
+      tags: true
+      branch: main
+      repo: twilio/flex-plugin-builder
+      node_js: '10'
```

Comment:
when do we get to node 14

Reply:
We already do test https://github.com/twilio/flex-plugin-builder/blob/v4-beta/.travis.yml#L5. This is for publishing (which must be done with one version only, otherwise you'll get duplicate versions being published). It's using Node 10 because that is still the most popular version.